### PR TITLE
footer module

### PR DESCRIPTION
**Pull request by flynnduism**
* move footer to a module
* port over footer css
* include footer within the dashboard view

### CODE REVIEW

**Comment by flynnduism** [comment]
Can you review #140 first?
It adds font styles which I can then apply on top of this one.

--

Closes #116.

**Comment by flynnduism** [comment]
Ready for review 🌮

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <div class="row">
   <app-sidebar></app-sidebar>
-  <div class="column" id="main">
-    <router-outlet></router-outlet>
+
+  <div class="cell flex-container flex-dir-column">
+    <div class="column flex-child-auto" id="main">
+      <router-outlet></router-outlet>
+    </div>
+    <div class="column flex-child-shrink" id="footer">
```
Comment (seanknox) [inline src/app/app.component.html]: Move this div into the footer component HTML, so the resulting app.component.html looks like:

```html
<div class="row">
  <app-sidebar></app-sidebar>

  <div class="cell flex-container flex-dir-column">
    <div class="column flex-child-auto" id="main">
      <router-outlet></router-outlet>
    </div>
    <app-footer></app-footer>
  </div>
</div>

```

**Comment by flynnduism** [comment]
Thanks @seanknox 

- [ ] footer width should fill a larger screen (likely this is an issue with #main too)
- [ ] fix `ng test`
- [ ] adjust footer div nested

Microsoft logo was removed before 0.1.0, it's by design since Kashti is not a direct Microsoft product or service.

**Comment by seanknox** [comment]
Looks good now! Just need to fix the tests and LGTM.

**Comment by seanknox** [comment]
:shipit: ⚓️

**Pull request merged**